assets/css/availability-table.scss

```diff
@@ -75,3 +75,15 @@ table.availability-table {
     color: white;
   }
 }
+
+@media screen and (min-width: 1001px) {
```

Comment:
Why are you update the user availability table, as this PR is related to the planning page ?

Reply:
My bad, I put the rules in the wrong place, fix in in progress.

Reply:
Those rules are now removed, all the rules needed are in planning.scss.